ReactiveCocoa/Swift/Property.swift

```diff
@@ -35,195 +35,186 @@ public protocol MutablePropertyType: class, PropertyType {
+	@warn_unused_result(message:"Did you forget to use the composed property?")
 	public func skipRepeats() -> AnyProperty<Value> {
 		return lift { $0.skipRepeats() }
 	}
 }
 
-extension PropertyType where Value: PropertyType {
+extension PropertyProtocol where Value: PropertyProtocol {
 	/// Flattens the inner properties sent upon `self` (into a single property),
 	/// according to the semantics of the given strategy.
-	@warn_unused_result(message="Did you forget to use the composed property?")
-	public func flatten(strategy: FlattenStrategy) -> AnyProperty<Value.Value> {
+	@warn_unused_result(message:"Did you forget to use the composed property?")
+	public func flatten(_ strategy: FlattenStrategy) -> AnyProperty<Value.Value> {
 		return lift { $0.flatMap(strategy) { $0.producer } }
 	}
 }
 
-extension PropertyType {
+extension PropertyProtocol {
 	/// Maps each property from `self` to a new property, then flattens the
 	/// resulting properties (into a single property), according to the
 	/// semantics of the given strategy.
-	@warn_unused_result(message="Did you forget to use the composed property?")
-	public func flatMap<P: PropertyType>(strategy: FlattenStrategy, transform: Value -> P) -> AnyProperty<P.Value> {
+	@warn_unused_result(message:"Did you forget to use the composed property?")
+	public func flatMap<P: PropertyProtocol>(_ strategy: FlattenStrategy, transform: (Value) -> P) -> AnyProperty<P.Value> {
 		return lift { $0.flatMap(strategy) { transform($0).producer } }
 	}
 
 	/// Forwards only those values from `self` that have unique identities across the set of
 	/// all values that have been seen.
 	///
 	/// Note: This causes the identities to be retained to check for uniqueness.
-	@warn_unused_result(message="Did you forget to use the composed property?")
-	public func uniqueValues<Identity: Hashable>(transform: Value -> Identity) -> AnyProperty<Value> {
+	@warn_unused_result(message:"Did you forget to use the composed property?")
+	public func uniqueValues<Identity: Hashable>(_ transform: (Value) -> Identity) -> AnyProperty<Value> {
 		return lift { $0.uniqueValues(transform) }
 	}
 }
 
-extension PropertyType where Value: Hashable {
+extension PropertyProtocol where Value: Hashable {
 	/// Forwards only those values from `self` that are unique across the set of
 	/// all values that have been seen.
 	///
 	/// Note: This causes the values to be retained to check for uniqueness. Providing
 	/// a function that returns a unique value for each sent value can help you reduce
 	/// the memory footprint.
-	@warn_unused_result(message="Did you forget to use the composed property?")
+	@warn_unused_result(message:"Did you forget to use the composed property?")
 	public func uniqueValues() -> AnyProperty<Value> {
 		return lift { $0.uniqueValues() }
 	}
 }
 
 /// Combines the values of all the given properties, in the manner described by
 /// `combineLatest(with:)`.
-@warn_unused_result(message="Did you forget to use the property?")
```

Comment:
I'm ok with removing this since the message didn't add that much and it's the default now.